docs/source/reference/kubernetes/kubernetes-setup.rst

```diff
@@ -483,7 +304,15 @@ Note that this dashboard can only be accessed from the machine where the ``kubec
     `Kubernetes documentation <https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/>`_
     for more information on how to set up access control for the dashboard.
 
+
 Troubleshooting Kubernetes Setup
 --------------------------------
 
 If you encounter issues while setting up your Kubernetes cluster, please refer to the :ref:`troubleshooting guide <kubernetes-troubleshooting>` to diagnose and fix issues.
+
+
+.. toctree::
+   :hidden:
+
+   kubernetes-deployment
+   kubernetes-ports
```

Comment:
How about reduce the length a bit, so that it does change lines in the TOC?
```suggestion
   Exposing Services <kubernetes-ports>
```

Also, would be nice to change the title in TOC for `Getting Started on Kubernetes` to `Getting Started`?